apps/console/src/features/email-providers/pages/email-providers.tsx

```diff
@@ -440,7 +440,7 @@ const EmailProvidersPage: FunctionComponent<EmailProvidersPageInterface> = (
             pageHeaderMaxWidth={ true }
             backButton={ {
                 onClick: handleBackButtonClick,
-                text: "Go back to Email Templates"
+                text: "Go back to notification channels"
```

Comment:
Shall we add translations to this?

Reply:
fixed: 6494aac1141609560b62f9b1294241d37ef00c95